.drone.yml

```diff
@@ -174,7 +174,7 @@ matrix:
       TEST_SUITE: phpunit
       DB_TYPE: sqlite
 
-    - PHP_VERSION: 7.2
```

Comment:
why was this necessary?

Reply:
Because the matrix entry below already has PHP 7.2 and was identical.

Reply:
Cool - THX